src/internal/setup_impl.c

```diff
@@ -777,6 +777,26 @@ void OSHMPI_initialize_thread(int required, int *provided)
     OSHMPI_CALLMPI(MPI_Comm_rank(OSHMPI_global.comm_world, &OSHMPI_global.world_rank));
     OSHMPI_CALLMPI(MPI_Comm_group(OSHMPI_global.comm_world, &OSHMPI_global.comm_world_group));
 
```

Comment:
Can we remove `OSHMPI_global.comm_world` and use `OSHMPI_global.team_world->comm`?

Reply:
Yes. Let me update that.